lib/admin/WP_Auth0_Admin_Generic.php

```diff
@@ -213,6 +234,21 @@ protected function render_field_description( $text ) {
 		printf( '<div class="subelement"><span class="description">%s.</span></div>', $text );
 	}
 
+	/**
+	 * Check if the setting is provided by a constant and indicate.
+	 *
+	 * @param string $input_name - Input name for the field, used as option key.
+	 */
+	protected function render_const_notice( $input_name ) {
+		if ( $this->options->has_constant_val( $input_name ) ) {
```

Comment:
this check is already done on some of the `render_const_notice` calls. Decide whether you'd like to do this here or outside, but don't check twice... it's an overkill. e.g. call


```php
protected function render_radio_buttons( array $buttons, $id, $input_name, $curr_value ) {
		if ( $field_is_const = $this->options->has_constant_val( $input_name ) ) {
			$this->render_const_notice( $input_name );
		}
```